treeherder/etl/classification_loader.py

```diff
@@ -88,3 +98,18 @@ def get_push(self, task_route):
             raise
 
         return push
+
+    def autoclassify_failures(self, failures):
+        autoclassified_intermittent = FailureClassification.objects.get(
+            name="autoclassified intermittent"
+        )
+        for tasks in failures.values():
+            # Keeping only the tasks that should be autoclassified
+            for task in [t for t in tasks if t.get("autoclassify")]:
```

Comment:
Nit: I would prefer to read `for task in tasks:` then simply check the `autoclassify` argument inside the loop. It will be more readable